test/cql-pytest/test_aggregate.py

```diff
@@ -1,13 +1,14 @@
-# Copyright 2023-present ScyllaDB
+# Copyright 2021-present ScyllaDB
```

Comment:
Why was copyright changed?

Reply:
Or maybe this is just git generating strange diffs?

Reply:
As I explained in the commit message, I combined two test files, one supposedly about min() and max() and the second about count(), into one test file about all the aggregations (the above and also sum()). The copyright I used for the new file was the earlier of the two. Git decided arbitrarily to consider this patch a "rename" of one of the files, which explains what the patch looks like.

Reply:
Right, I thought this was the case, git does generate counter-intuitive diffs sometimes.